test/Project.toml

```
@@ -3,4 +3,4 @@ Aqua = "4c88cf16-eb10-579e-8560-4a9242c79595"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 
 [compat]
-Aqua = "0.7"
+Aqua = "0.8"
```

```suggestion
Aqua = "0.8.2"
```
this should work now around https://github.com/JuliaTesting/Aqua.jl/pull/240#discussion_r1395786043